tests/swagger_generation/test_marshmallow_to_swagger.py

```diff
@@ -135,27 +136,49 @@ def test_primitive_types(self):
             ),
             (m.fields.Integer(validate=lambda value: True), {"type": "integer"}),
         ]:
+            with self.subTest(field=field):
+                self._test_primatives(field=field, result=result)
 
-            class Foo(m.Schema):
-                a = field
+    def test_primative_types_newer(self):
+        marsh_version_tuple = tuple(
+            [int(digit) for digit in version("marshmallow").split(".")]
+        )
+        if marsh_version_tuple < (3, 18, 0):
+            self.skipTest("marshmallow version older than 3.18.0")
+        for field, result in [
+            (
+                me.EnumField(StopLight),
+                {"enum": ["green", "yellow", "red"], "type": "string"},
+            ),
+            (
+                me.EnumField(StopLight, by_value=True),
+                {"enum": [1, 2, 3], "type": "integer"},
+            ),
+        ]:
```

Comment:
Tests for `m.EnumField` are already encompassed in `test_primative_types`. Did you mean to test a different field?

Reply:
yes, i meant to switch these to EnumSymbol and EnumValue, but i need to redo this again, because they actually publish a regular Enum field on its own.